src/odemis/gui/cont/tabs.py

```diff
@@ -4766,12 +4767,11 @@ def _set_sem_alignment_mode(self):
                                                      self._FIB_view_and_control)
 
         # Adjust the overlay to a normal crosshair in the FIB viewport
-        for viewport in self.panel.pnl_two_streams_grid.viewports:
-            if viewport.view.stream_classes is acqstream.FIBStream:
-                for FIB_line_overlay in viewport.canvas.view_overlays:
-                    if isinstance(FIB_line_overlay, CenteredLineOverlay):
-                        FIB_line_overlay.shape = CROSSHAIR
-                        break
+        fib_viewport = self._FIB_view_and_control["viewport"]
+        for FIB_line_overlay in fib_viewport.canvas.view_overlays:
+            if isinstance(FIB_line_overlay, CenteredLineOverlay):
+                FIB_line_overlay.shape = CROSSHAIR
+                break
 
```

Comment:
It is actually, whenever z mode alignment is set a horizontal line is used and otherwise a crosshair.
```
if isinstance(FIB_line_overlay, CenteredLineOverlay):
    FIB_line_overlay.shape = HORIZONTAL_LINE
    break
```
This part should also be updated such as https://github.com/delmic/odemis/pull/1963#discussion_r845027466 actually.

Reply:
Sorry, I missed that part. It's all fine then!

Reply:
it is actually changed. So I leave it as is.